support/azure/azure-kubernetes/create-upgrade-delete/error-code-vhdfilenotfound.md

```diff
@@ -30,20 +30,26 @@ Under rare circumstances, the 124 exit code for the Azure Virtual Machine Scale
 | `K8SAPIServerConnFailVMExtensionError`      | 51                |
 | `K8SAPIServerDNSLookupFailVMExtensionError` | 52                |
 
```

Comment:
This error occurs if a connectivity issue exists between your AKS cluster and the required Azure endpoints, such as `mcr.microsoft.com` or `acs-mirror.azureedge.net`.